plugins-base/XSFeatureNetworkReset.py

```diff
@@ -335,11 +343,16 @@ def Commit(self):
 			finally:
 				f.close()
 
-		# Construct bridge name for management interface based on convention
-		if self.device[:3] == 'eth':
-			bridge = 'xenbr' + self.device[3:]
-		else:
-			bridge = 'br' + self.device
+		renamed = os.path.exists(interface_rename_dir)
+
+		if renamed:
+			# Construct bridge name for management interface based on convention
+			if self.device[:3] == 'eth':
+				bridge = 'xenbr' + self.device[3:]
+			else:
+				bridge = 'br' + self.device
+		# Else, the device name will not be like eth<N>. And networkd will
+		# determine the bridge name.
 
 		# Ensure xapi is not running
 		os.system('service xapi stop >/dev/null 2>/dev/null')
```

Comment:
not related to your PR, but 
`service` command is no longer valid in XS9, 
is it possible for you to update it to `systemctl stop xapi`

Reply:
well, let's focus on your task, 
We will fix it latter in another PR